lib/storekit/receipt_response.rb

```diff
@@ -0,0 +1,27 @@
+module StoreKit
+  class ReceiptResponse
```

Comment:
The naming convention you've selected just break the understandability because you actually have named the response from itunes with name of response's object. That is wrong.